.github/workflows/quality-check.yaml

```diff
@@ -100,7 +100,7 @@ jobs:
           cache: 'gradle'
       - name: Run Linter
         run: |
-          ./gradlew --quiet --console=plain spotlessApply
+          ./gradlew --quiet --console=plain spotlessCheck
```

Comment:
Yea and this one because we probably don't wanna apply styling changes in the CI but only check?

Reply:
Makes a lot of sense!